scripts/react_native_pods.rb

```diff
@@ -46,6 +46,9 @@ def use_react_native! (
 
   CodegenUtils.clean_up_build_folder(app_path, $CODEGEN_OUTPUT_DIR)
 
+  # We are relying on this flag also in third parties libraries to proper install dependencies.
+  # Better to rely and enable this environment flag if the new architecture is turned on using flags.
+  ENV['RCT_NEW_ARCH_ENABLED'] = new_arch_enabled ? "1" : "0"
```

Comment:
I'm seeing we use single quotes when there are no string interpolations.
It's okay if it doesn't work when we use single quotes here.